.githooks/pre-commit

```diff
@@ -1,34 +1,50 @@
 #!/usr/bin/env bash
 
-check_rs=0
-check_ts=0
+debug=0                         # Set to 1 to enable debugging, 0 to disable.
+do_check_rs=0
+do_check_ts=0
 
-declare -a extra_flags=("--" "--cached")
+check_rs_cmd="cargo make check-rs-format"
+check_ts_cmd="cargo make check-ts-format"
+if (( debug == 1 )); then
+    check_rs_cmd="echo check_rs ran"
```

Comment:
I don't think you want to keep that in here, I guess it's for testing?

Reply:
We want this in IMO because next time someone wants to modify this script the testing capability is in there ready.  (I added this because of all the iterations so far).